packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -250,6 +250,17 @@ type Link implements Widget {
   label: String!
   iconURL: String
   url: String!
+  display: String!
```

Comment:
In a similar fashion as Label, I'll also remove references to `display` and `displayExpression`.